analysis/topology/uniform_space.lean

```diff
@@ -1558,12 +1582,31 @@ tendsto_prod_uniformity_fst
 lemma uniform_continuous_snd [uniform_space α] [uniform_space β] : uniform_continuous (λp:α×β, p.2) :=
 tendsto_prod_uniformity_snd
 
-lemma uniform_continuous.prod_mk [uniform_space α] [uniform_space β] [uniform_space γ]
+variables [uniform_space α] [uniform_space β] [uniform_space γ]
+lemma uniform_continuous.prod_mk 
   {f₁ : α → β} {f₂ : α → γ} (h₁ : uniform_continuous f₁) (h₂ : uniform_continuous f₂) :
   uniform_continuous (λa, (f₁ a, f₂ a)) :=
 by rw [uniform_continuous, uniformity_prod]; exact
 tendsto_inf.2 ⟨tendsto_vmap_iff.2 h₁, tendsto_vmap_iff.2 h₂⟩
 
+lemma uniform_continuous.prod.partial1 {f : α × β → γ} (h : uniform_continuous f) :
```

Comment:
Are these two lemmas really necessary? They look very specific to me.

If you want to keep them:
* naming is more like: `uniform_continuous.prod_mk_left`, `uniform_continuous.prod_mk_right`
* indentation: new line after `:=`, move the `∀ b` to an parameter (i.e. before the `:`) and hence remove the lambda.